core/integration/module_test.go

```diff
@@ -5119,6 +5119,93 @@ func (t *Test) IgnoreDirButKeepFileInSubdir(
 	})
 }
 
+func (ModuleSuite) TestCombinedOutput(ctx context.Context, t *testctx.T) {
+	tcs := []struct {
+		sdk    string
+		source string
```

Comment:
IMO, this API should be tested as part of `ContainerSuite`.

We also don't need to test each SDK separately, since it's an API entirely in the engine, we can just use the client available in `connect` directly.

Reply:
Sure, I'll update that 👍